### PR TITLE
new tenet: rpc-responses-display-uniformly

### PR DESCRIPTION
**Pull request by mullikine**

### CODE REVIEW

**Comment by mullikine** [comment]
# context
It's not possible to write CLQL for protobuf files, so in order to write this tenet we must check the generated ~.go~ code.

This checks fields which will turn into the ~.go~.

# summary of guidelines
The guidelines here:

https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md

Menion the following.

Additionally, in order to maintain a uniform display of the RPC responses
rendered by `lncli`, all added or modified `protof` definitions, _must_ attach
the proper `json_name` option for all fields. An example of such an option can
be found within the definition of the `DebugLevelResponse` struct:

    protobuf
    message DebugLevelResponse {
        string sub_systems = 1 [ json_name = "sub_systems" ];
    }

Notice how the `json_name` field option corresponds with the name of the field
itself, and uses a `snake_case` style of name formatting. All added or modified
`proto` fields should adhere to the format above.

File: tenets/codelingo/lightning-network-daemon/rpc-responses-display-uniformly/example.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package main
+
+import proto "github.com/golang/protobuf/proto"
+
+import "fmt"
+
+type TestPB struct {
+	CorrectName []byte `protobuf:"bytes,2,opt,name=correct_name,proto3" json:"correct_name,omitempty"`
+	IncorrectName []byte `protobuf:"bytes,2,opt,name=incorrectname,proto3" json:"incorrect_name,omitempty"`
+	AnotherIncorrectName []byte `protobuf:"bytes,2,opt,name=incorrect_name,proto3" json:"incorrect_name,omitempty"`
```
Comment (rstorr) [inline tenets/codelingo/lightning-network-daemon/rpc-responses-display-uniformly/example.go]: name == json here isn't that correct?

File: tenets/codelingo/lightning-network-daemon/rpc-responses-display-uniformly/codelingo.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    flows:
+      codelingo/docs:
+        title: RPC responses display uniformly with snake case
+        body: |
+      codelingo/review:
+        comment: The protobuf name and json_name must be snake_case. Please change their values in the corresponding .proto file.
+    query: |
+      import codelingo/ast/go
+      
+      go.file(depth = any):
```
Comment (rstorr) [inline tenets/codelingo/lightning-network-daemon/rpc-responses-display-uniformly/codelingo.yaml]: remove go.file.

File: tenets/codelingo/lightning-network-daemon/rpc-responses-display-uniformly/codelingo.yaml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    flows:
+      codelingo/docs:
+        title: RPC responses display uniformly with matching ident
+        body: |
+      codelingo/review:
+        comment: type field does not match the protobuf name and json_names defined here. Please change their values to match in the corresponding .proto file.
+    query: |
+      import codelingo/ast/go
+      
+      go.file(depth = any):
```
Comment (rstorr) [inline tenets/codelingo/lightning-network-daemon/rpc-responses-display-uniformly/codelingo.yaml]: remove go.file

File: tenets/codelingo/lightning-network-daemon/rpc-responses-display-uniformly/codelingo.yaml [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    flows:
+      codelingo/docs:
+        title: RPC responses display uniformly with fields equal
+        body: |
+      codelingo/review:
+        comment: protobuf name and json_name are not equal here. Please change their values to match in the corresponding .proto file.
+    query: |
+      import codelingo/ast/go
+      
+      go.file(depth = any):
```
Comment (rstorr) [inline tenets/codelingo/lightning-network-daemon/rpc-responses-display-uniformly/codelingo.yaml]: remove go.file

Comment (mullikine) [reply tenets/codelingo/lightning-network-daemon/rpc-responses-display-uniformly/codelingo.yaml]: Unfortunately, are not able to remove as-is, but will try after squashing the tenets..
https://trello.com/c/sLnkYGsp/455-gofile-sometimes-not-implicit

**Comment by rstorr** [comment]
Update https://github.com/codelingo/codelingo/blob/master/registry/tenets.yaml

**Pull request merged**